gensrc/thrift/PlanNodes.thrift

```diff
@@ -420,6 +420,7 @@ struct TSchemaScanNode {
   11: optional Types.TUserIdentity current_user_ident   // to replace the user and user_ip
 }
 
+// If you find yourself changing this code, see also TLakeScanNode
```

Comment:
better to add some comment in `OlapScanNode.toThrift` function as well

Reply:
Updated, PTAL